examples/cpp/route_guide/helper.cc

```diff
@@ -43,6 +44,8 @@ ABSL_FLAG(std::string, db_path, "route_guide_db.json", "Path to db file");
 namespace routeguide {
 
 std::string GetDbFileContent(int argc, char** argv) {
+  // Command-line flags should be parsed at startup
+  absl::ParseCommandLine(argc, argv);
```

Comment:
actually, let's move this to each main. That way we won't have to pass in argc and argv.

Reply:
In previous version, I see absl::ParseCommandLine was executed in each main of client and server and both of client and server can't parse db_path arguments from cmdlines. Let's try it!
It should be placed in same file helper.cc.